plugins/header_rewrite/parser.cc

```diff
@@ -133,14 +152,15 @@ Parser::preprocess(std::vector<std::string> tokens)
       std::string s = tokens[0].substr(2, tokens[0].size() - 3);
 
       _op = s;
-      if (tokens.size() > 2 && (tokens[1][0] == '=' || tokens[1][0] == '>' || tokens[1][0] == '<')) { // cond + (=/</>) + argument
+      if (tokens.size() > 2 && (tokens[1][0] == '=' || tokens[1][0] == '>' || tokens[1][0] == '<')) {
+        // cond + [=<>] + argument
         _arg = tokens[1] + tokens[2];
       } else if (tokens.size() > 1) {
+        // This is for the regular expression, which for some reason has its own handling?? ToDo: Why ?
```

Comment:
The comment? It's what it says, that case is for the regular expression case, I kinda wish we had treated / as the other operators (=, < and >), but we do not. And I don't think this is the time to change that.